### DISCUSSION TITLE
Favoriser l'accès aux itinéraires via des mobilités alternatives

### DISCUSSION THREAD

**Discussion by babastienne** [GeotrekCE/Geotrek-admin#3996] [Ideas]
Ne plus prendre la voiture et inciter les visiteurs à venir via d'autres modes de mobilités est un enjeu majeur pour les territoires. Geotrek, qui a toujours souhaité s'inscrire dans une démarche durable, de sensibilisation et d'éco-responsabilité devrait selon moi mettre beaucoup plus en avant l'accès aux départ des itinéraires via des mobilités douces ou actives.

Aujourd'hui, voici ce que donne une fiche "correctement" remplie en terme d'accès : 

![image](https://github.com/GeotrekCE/Geotrek-admin/assets/16348048/1bdef78b-62ad-4ebc-8f3c-f1a07b2c6f5d)

Pour moi il faudrait à minima repenser une partie de l'interface et des fonctionnalités de Geotrek pour permettre de mettre beaucoup plus en avant les accès autres que motorisés : 
- Sur la carte, permettre de mettre en avant les gares, parking vélo et lignes de bus, au même titre que le parking actuellement
- Proposer dans la liste des résultats des filtres pour rapidement filtrer les contenus accessibles via certains vecteurs (ça impliquerai côté Admin d'avoir un champ "moyen d'accès" où on pourrait sélectionner plusieurs moyens d'accès et remplir pour chacun un descriptif associé ?)

Mais on pourrait aller beaucoup plus loin, il y a plein de pistes à explorer : 
- Créer des connecteurs automatiques pour récupérer les flux des lignes de transport en commun et les afficher sur la carte ou même générer une description associée avec les horaires en fonction de la période de consultation de l'utilisateur
- Ajouter un [widget](https://www.sncf-connect.com/outils/widget) pour permettre à l'utilisateur de programmer son trajet en train dès lors qu'une gare est à moins de x km du départ de l'itinéraire
- Ajouter un widget pour sensibiliser à l'empreinte carbone émise selon la mobilité choisie pour se rendre à l'itinéraire via un widget dédié (cf [lowtrip](https://github.com/XavB64/lowtrip) ?)
- etc.

C'est évidement des pistes mais j'aimerai qu'on puisse creuser ce sujet car c'est un réel enjeu important à la fois pour des raisons de sensibilisation et d'impact sur les espaces naturels, mais aussi pour répondre à des besoins de plus en plus présent chez les visiteurs et chez les territoires qui cherchent un outil de gestion / promotion territorial.

Il faut que Geotrek pusse contribuer à modifier les habitudes de déplacement, en proposant une interface tellement intuitive qu'elle incite un utilisateur qui n'y avait même pas songé à ne pas prendre sa voiture.

----

Aujourd'hui on constate que certains territoires on déjà tenté de répondre à ce besoin via diverses solutions : 
- le [PNR Millevaches](https://rando-millevaches.fr/) propose une catégorie dédiée aux randonnées accessibles en train sur sa page d'accueil, mais cela oblige à surcharger la barre d'activités. De plus pour mettre en avant la gare ils ont du ajouter une autre carte dans la description de la [fiche randonnée](https://rando-millevaches.fr/trek/735539-Les-villages) car Geotrek ne permet pas la mise en avant d'une gare sur la carte ...
- le [PNR des Grands Causses](https://rando.parc-grands-causses.fr/search?practices=12&accessibilities=20) a adopté une stratégie similaire, avec la possibilité de filtrer par accessibilité 'Rando-Train'. Mais il s'agit d'un contournement d'un champ plutôt initialement prévu pour l’accessibilité aux personnes en situation de handicap. 
- Pendant un temps le département du Gard avait listé les gares existantes sur le territoire ainsi que les arrêts de bus comme des "contenus touristiques" avec leur catégorie dédiée.

Ces exemples illustrent bien le fait qu'il y a un besoin non couvert actuellement dans Geotrek.

----

A travers cette discussion j'aimerai qu'on puisse imaginer le périmètre fonctionnel qu'on aimerait avoir autour de ce sujet des mobilité dans Geotrek : jusqu'où on souhaite aller ? quelles limites ? quelles implications techniques ?
Ensuite à partir de là si on s'accorde sur un périmètre et qu'on est capable d'estimer ce que ça engendre en terme de développements j'aimerai proposer aux territoires de la communauté Geotrek de participer s'ils le souhaite pour financer le développement de cette / ces évolutions.

**Comment by amandine-sahl** [comment]
Une démarche de promotion des itinéraires pédestres accessibles en transport en commun a été entrepris par le parc national des Cévennes. Elle se traduit dans Geotrek de la façon suivante : 
 - Les itinéraires ont tous été identifié via le thème transport en commun
 - Le champ transport en commun a été remplis avec le numéro de la ligne et les noms des arrêts de départ et d'arrivée
 
 Dans geotrek - rando
 - Un encart sur la page d’accueil permet de les filtrer (https://destination.cevennes-parcnational.fr/search?themes=24)
 - Le calculateur d'itinéraire liO a été ajouté aux fiches des itinéraires


![image](https://github.com/user-attachments/assets/11e40f9e-6697-4e43-9ab6-ff08df71e9cd)

![image](https://github.com/user-attachments/assets/a9272b51-2304-450d-b8d0-9744f9498f45)

**Reply by babastienne** [reply]
Merci @amandine-sahl pour ce retour d'expérience. 

C'est encore une autre manière de promouvoir les mobilités alternatives. Ca confirme bien que pour le moment le manque de structuration sur ce sujet implique que les territoires font chacun un peu à leur sauce (via les thèmes, via les étiquettes, etc.). 

Pour moi il faudrait de manière claire proposer une méthode pour promouvoir ces éléments. Le problème des étiquettes ou des thèmes c'est que c'est mélangé avec le reste et que c'est pas forcément très clair pour l'utilisateur final (intuitivement un utilisateur ne va pas aller chercher dans les liste des thèmes pour trouver les itinéraires accessibles en transport en commun, il faut que l'interface du site mette spontanément en avant l'information pour promouvoir la démarche).

Très chouette le calculateur d'itinéraires liO, c'est top ! A diffuser auprès des autres territoires pour leur donner des idées peut-être ?

**Reply by s-mazoyer** [reply]
Bonjour à toutes les deux,
Merci de cette discussion. 
Sur notre territoire, une vingtaine de boucles de rando pédestre sont accessibles avec le transport public routier régional et nous souhaitons diffuser cette information. Nous nous interrogeons sur la possibilité de faire figurer les points d'arrêts et les lignes de transport sur Geotrek avec mise à jour automatique (flux). Ces données transport sont disponibles au format GTFS. Auriez-vous une idée sur la possibilité de les intégrer dans Geotrek (POI, service,...) ? Ou un contact à me donner pour cela ?
Voici ensuite mon avis sur un aspect discuté plus haut : je trouve utile de pouvoir sélectionner dès le début d'une recherche de rando celles accessibles en transport en commun ("J'ai envie de randonner sans voiture, quelles sont les offres ?", plutôt que "J'ai envie de faire cette rando, je suis prête à y aller en voiture ou en TC, selon ce qui est possible"). Si on veut inciter ou faciliter la rando sans voiture, autant mettre en évidence les possibilités offertes.
A suivre...

**Reply by camillemonchicourt** [reply]
Une question qui se pose est quelles données sont à stocker au niveau de Geotrek-admin et lesquelles sont lues et affichées directement sur Geotrek-rando depuis des flux de sources externes.

**Reply by s-mazoyer** [reply]
Je  découvre seulement Geotrek et n'ai qu'une réponse limitée : pour l'usager, les données à afficher sont selon moi : sur la carte, les points d'arrêt à proximité de la rando (tampon) ; en commentaire, le nom des arrêts, les communes et lignes qui y passent. Puis, le lien vers le calculateur d'itinéraires du réseau de transport et sa page des fiches horaires en pdf (parfois, une fiche horaire est plus facile à utiliser qu'un calculateur).

**Reply by babastienne** [reply]
Bonjour @s-mazoyer :wave: 

Quelques réflexions par rapport à ton message : 
- Faire figurer et mettre à jour les points d'arrêts : ça serait possible relativement facilement en exploitant par exemple la notion actuelle de service dans Geotrek. Il faudrait juste implémenter un parser pour synchroniser à une intervalle régulière les données afin de les garder à jour. On pourrai même jouer sur les catégories de contenus touristique pour préciser par exemple les type de transport et les numéro de lignes. A ma connaissance ça n'a pas été fait encore (un peu avec les arrêt de car Lio dans le département du Gard mais c'était perfectible)
- Pour les lignes de transport c'est plus compliqué mais on pourrait imaginer créer une catégorie de contenus touristique aussi qui gère les lignes (puisque théoriquement GTA est capable de gérer des contenus touristiques de type ligne, et que ça marche sur Geotrek-Rando)
- On avait regardé il y a un peu plus d'un an pour un territoire la possibilité d'exploiter des flux GTFS. Au final le projet ne s'était pas fait car nous n'avions pas remporté le marché mais en soit notre étude avait jugée que c'était tout à fait faisable d'exploiter ces flux. Là où c'était peut-être un peu plus difficile c'était sur la notion de temps réel (où est le bus ? dans combien de temps arrive-t-il ?). Mais si c'est juste pour récupérer les lignes et les fiches horaires c'est jouable.

Comme pointé par @camillemonchicourt, ça lève tout de même la question de savoir où et comment gérer/stocker cette donnée. Est-ce le rôle de Geotrek-admin, qui est plutôt un outil de gestion d'itinéraires, de stocker des données de transports en commun ? Si on estime que non alors ça serait plutôt à Geotrek-Rando de lire ces données en temps réel via une API, mais ça lève des questions alors sur les performances, la disponibilité, etc. > C'est un sujet à creuser un peu plus en détail je pense.

Après il y a aussi les questions de ce qu'on cherche à mettre en avant : si c'est uniquement les infos listées dans le commentaire précédent ça me semble jouable. Si c'est intégrer à Geotrek un calculateur d'itinéraires en transports en commun, une alerte sur le prochain passage d'un bus ou autre ça commence à devenir compliqué et nécessiter du développement. 

_____

> je trouve utile de pouvoir sélectionner dès le début d'une recherche de rando celles accessibles en transport en commun : "J'ai envie de randonner sans voiture, quelles sont les offres ?"

Oui je suis 100% d'accord

______

Le sujet m'intéresse. Si tu veux on peut essayer de prévoir un temps ensemble pour réfléchir à ce qu'il serait possible de mettre en place pour essayer de répondre à votre besoin dans Geotrek ? En l'état je pense qu'on peut partir avec ce qui existe déjà (même si c'est perfectible mais c'est déjà mieux que rien) avec toutefois surement un développement à faire pour prendre en compte les flux GTFS dans Geotrek. Et une fois qu'on a les lignes / services de transport qui sont intégrés et qui remontent voir comment faire remonter simplement avec un filtre dans Geotrek-Rando les contenus à proximité des ces transports.

**Reply by s-mazoyer** [reply]
Merci bcp à toutes les deux pour votre réactivité et les propositions. Je vous retrouve après mes congés qui démarrent... now ! ;)
A bientôt.

**Reply by camillemonchicourt** [reply]
Oui c'est un sujet qui nous intéresse aussi ainsi que pas mal d'utilisateurs Geotrek.
Mon à priori serait de faire le max de choses côté Geotrek-rando pour être dynamique et stocker le moins de choses possibles côté dynamiques, interroger les horaires en temps réel, etc... 
Mais faut bien étudier le truc. 
Il faut certainement stocker des éléments minimaux côté Geotrek-admin (genre les lignes et/ou les points d'arrêt, les randos accessibles en transport avec un ID universel) et interroger les données dynamiques (horaires...) directement au niveau de Geotrek-rando.
Mais à étudier.

Je pense que ça mérite un petit atelier technique / café Geotrek pour creuser ensemble le besoin et les premières pistes techniques.

**Reply by raphaeldoisy** [reply]
Bonjour à toutes et à tous,

Je rejoins le besoin de faciliter la visualisation des transports en communs pour mieux planifier sa rando.
Je note l'idée que ça mérite en effet un atlier technique / café Geotrek pour creuser le sujet car ayant fait ce genre de recherche étant randonneur et voulant utiliser que des transports en communs, l'accès à l'information n'était pas évident. Comme vous l'avez dit, Geotrek-admin pourrait potentiellement accueillir ces données ou sinon Geotrek-rando via API.

**Reply by s-mazoyer** [reply]
Bonjour,
Comment être informée des ateliers techniques / cafés Geotrek ? Dans ce fil de discussion ?

**Reply by raphaeldoisy** [reply]
Bonjour @s-mazoyer ,

Il existe une liste de diffusion dans laquelle des informations sur les cafés geotrek, la vie de la communauté circulent ( des questions d'ordre général, des propositions de cafés geotrek, d'ateliers techniques, les nouvelles versions de Geotrek-admin, Geotrek-rando...)
Je t'invite donc à t''inscrire à ce groupe: https://groups.google.com/g/geotrek-fr
Si tu rencontres des difficultés, n'hésite pas à revenir vers nous.

**Comment by babastienne** [comment]
Bon, on se heurte à un autre problème ... En regardant certains widget (réservation de train via sncf connect, réservation de train via RER lio, etc.) on se rend compte que pour pas mal d'entre eux l'accès au widget n'est pas libre, qu'il faut un toke, une demande d'autorisation, etc. Bref pas très libre tout ça, donc pas forcément simple ...

**Comment by babastienne** [comment]
La plate-forme CampToCamp propose désormais sur toutes ces fiches d'itinéraires un module bien visible pour faciliter (et mettre en avant) l'accès en transports en communs ([exemple](https://www.camptocamp.org/routes/311302/fr/quie-de-sinsat-la-pelle-en-attendant-marco)) : 

<img width="1191" height="649" alt="image" src="https://github.com/user-attachments/assets/37882032-6211-461e-8b7a-15b65ebf74bf" />

**Comment by s-mazoyer** [comment]
Bonjour Bastien,

Intéressant, merci.

Un autre exemple : Mountain wilderness promeut aussi la montage sans voiture [Changer d’approche - Sans voiture](https://www.changerdapproche.org/les-bons-plans/sans-voiture.html)

Bonne journée,

Sylvie MAZOYER
Direction Des Transports Scolaires et Interurbains
Service Réseau Vaucluse‑Bouches‑du‑Rhône
